test/e2e/components/vl-cookie-consent.js

```diff
@@ -27,7 +27,7 @@ class VlCookieConsent extends VlElement {
     }
```

Comment:
Misschien niet de moeite, maar `getOptIn` kan eventueel gebruik maken van `getOptIns` om te filteren, maar de winst is hier beperkt.

Reply:
Ja, dat kan inderdaad. Ik denk dat het qua performantie misschien iets sneller is om het te houden as-is, maar dat zal waarschijnlijk maar marginaal beter zijn.

Deze opmerking heeft me wel doen inzien dat de vl-ui-checkbox e2e component momenteel wel nog een getter voor het label mist. Heb gevraagd aan @wautergu om dat in zijn ticket toe te voegen.